index.markdown

```diff
@@ -1,10 +1,75 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 
-# Big Test Energy
+# UWB Hacks 2020 Documentation
+
+![logo](https://cdn.discordapp.com/attachments/652802883531571210/668393411564404736/websiteDesign1.png)
```

Comment:
Add final logo when logo gets finalized

Reply:
I like where your head is at with this; however, I think that the logo should be added to the page's header when it is ready (see image below), instead of the body of the index page. This is a personal opinion though; feel free to change my mind. 

The task to add the logo & event branding is documented in this issue: #4 

![docs_page_banner](https://user-images.githubusercontent.com/29528527/73616213-6a267300-45c5-11ea-935c-7c92efc703eb.png)

Reply:
Also, instead of linking to external platforms, we can add images/content to this repo so that all the images are in one place. The advantage of this is we don't have to worry about losing access to content if it is deleted from other platforms.